custom.d.ts

```diff
@@ -0,0 +1,13 @@
+declare module 'ol/format/GeoJSON';
+declare module 'ol/layer/Tile';
+declare module 'ol/layer/Vector';
+declare module 'ol/source/OSM';
+declare module 'ol/source/Vector';
+// will be declared in geostyler-openlayers-parser
+// declare module 'ol/Feature';
+declare module 'ol/Map';
+declare module 'ol/View';
+declare module 'ol/geom/LineString';
+// will be declared in geostyler-openlayers-parser
+// declare module 'ol/geom/Point';
```

Comment:
I had to include both declarations to get things running. Maybe this was related to a `npm link` to the `geostyler-openlayers-parser` in your local checkout?

Reply:
Fixed in bf54595